pkg/sql/crdb_internal.go

```diff
@@ -7542,3 +7544,68 @@ CREATE TABLE crdb_internal.node_memory_monitors (
 		return p.extendedEvalCtx.ExecCfg.RootMemoryMonitor.TraverseTree(monitorStateCb)
 	},
 }
+
+var crdbInternalShowTenantCapabilitiesCache = virtualSchemaTable{
+	comment: `eventually consisten in-memory tenant capability cache for this node`,
+	schema: `
+CREATE TABLE crdb_internal.show_tenant_capabilities_cache (
```

Comment:
I think `show_` made some sense when it was a function, but I'm not sure it makes sense as a table name.

Reply:
I'll rename this to `node_tenant_capabilities_cache` to be consistent with the other node-specific tables.